### DISCUSSION TITLE
Why do I need to enable Javascript? --solved--

### DISCUSSION THREAD

**Discussion by Dagur** [less/less.js#3779] [General]
Hi

I'm upgrading *less* from 2.7.1 to 4.1.3

I had to make some minor changes to my less files but mostly it was fine. There's one error that I couldn't get rid of though which was **Inline JavaScript is not enabled. Is it set in your options?**.

I changed the webpack config to enable javascript and then everything compiled without complaints but why did I need to do that if it worked fine before? We don't have any inline javascript as far as I know, only a few macros.

**Comment by Dagur** [comment]
I figured this out. There was a macro that was using a regex (and therefor javascript). The error reporting from less wasn't great because it complained where a macro that used this macro was being used.

**Comment by iChenLei** [comment]
Maybe you can check the Changelog of less.js ?

**Comment by Dagur** [comment]
I just looked at https://github.com/less/less.js/releases which mentions of couple of breaking changes but not this. 

Apparently javascript was enabled by default until version 3
